peglib.h

```diff
@@ -2188,11 +2353,7 @@ class ParserGenerator
                                seq(lit("\\x"), cls("0-9a-fA-F"), opt(cls("0-9a-fA-F"))),
                                seq(npd(chr('\\')), dot()));
 
-#if !defined(PEGLIB_NO_UNICODE_CHARS)
-        g["LEFTARROW"]  <= seq(cho(lit("<-"), lit(u8"←")), g["Spacing"]);
-#else
-        g["LEFTARROW"]  <= seq(lit("<-"), g["Spacing"]);
-#endif
+        g["LEFTARROW"]  <= seq(cho(lit("<-"), lit("\xe2\x86\x90")), g["Spacing"]);
```

Comment:
\xe2\x86\x90 is the UTF-8 representation of the left arrow. Now the source is pure ASCII and should work with any compiler. I hope.